### PR TITLE
Feature return loss

### PR DESCRIPTION
**Pull request by vertefra**
- Add a `loss` property to the base client to retrieve the `loss` matrix for the current model

Screenshot from 2024-07-15 17-02-20

### CODE REVIEW

File: qcog_python_client/qcog.py [hunk 1]
```diff
@@ -549,6 +591,11 @@ def status(self) -> TrainingStatus:
         )
 
         self.last_status = TrainingStatus(self.status_resp["status"])
+
+        # Try to set the _loss attribute if present
+        if self.status_resp.get("loss"):
+            self._loss = self.status_resp["loss"]
```
Comment (stand-by) [inline qcog_python_client/qcog.py:597]: what do you think of the following?
```
        if loss := self.status_resp.get("loss"):
            self._loss = loss
```

File: qcog_python_client/qcog.py [hunk 2]
```diff
@@ -112,6 +121,38 @@ def __init__(self) -> None:  # noqa: D107
         self._training_parameters: dict | None = None
         self._trained_model: dict | None = None
         self._inference_result: dict | None = None
+        self._loss: Matrix | None = None
+
+    @property
+    def loss(self) -> Matrix | None:
```
Comment (stand-by) [inline qcog_python_client/qcog.py:127]: Btw I just noticed that the output is the matrix. So maybe we should explain the shape and structure of this matrix in the docstring? what's along each axis etc

File: qcog_python_client/qcog.py (hunk 1 above)
Comment (vertefra) [reply qcog_python_client/qcog.py:597]: honestly, `_loss` can be `None`, I can just directly assign the value from the `get` method

**Review by stand-by — APPROVED** [review]
